tests/ui/abi/removed-wasm-abi.rs

```diff
@@ -1,4 +0,0 @@
-extern "wasm" fn test() {}
```

Comment:
why remove this test?

Reply:
It's a suggestion that was for migrating off a nightly feature, starting from last July.

The only users of the feature that I'm aware of are people implementing next-generation wasm toolchain components. So, the people working with us on the migration of wasm's `extern "C"` to match clang's. They... have known about this for some time, so it seems unlikely they will still benefit from a continued reminder into the future.

So, I removed the suggestion, as we don't usually keep around migration stuff for nightly things for more than a couple of releases, and it's been some.